content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md

```diff
@@ -79,75 +119,37 @@ To check the AD join progress, open the <i class="material-icons" aria-hidden="t
 TrueNAS displays any errors during the join process in the **Task Manager**.
 {{< /expand >}}
 
-When the import is complete, AD users and groups become available while configuring basic dataset permissions or an ACL with TrueNAS cache enabled (enabled by default).
+When the import completes, AD users and groups become available while configuring basic dataset permissions or an ACL with TrueNAS cache enabled (enabled by default).
 
-Joining AD also adds default Kerberos realms and generates a default `AD_MACHINE_ACCOUNT` keytab.
+Joining AD also adds default Kerberos realms and generates a default **AD_MACHINE_ACCOUNT** keytab.
 TrueNAS automatically begins using this default keytab and removes any administrator credentials stored in the TrueNAS configuration file.
 
-### Troubleshooting - Resync the Cache
+### Troubleshooting - Resyncing the Cache
 If the cache becomes out of sync or fewer users than expected are available in the permissions editors, resync it by clicking **Settings** in the **Active Directory** window and selecting **Rebuild Directory Service Cache**.
 
 If you are using Windows Server with 2008 R2 or older, try creating a **Computer** entry on the Windows server Organizational Unit (OU).
```

Comment:
Remove this portion. Server 2008R2 is EOL and we can't maintain docs for it.

Reply:
Removed